app/views/alert_mailer/alert_email.html.erb

```diff
@@ -1,27 +1,27 @@
 <h1>Energy Sparks alerts for <%= link_to @school.name, school_url(@school) %></h1>
 <p>You have been assigned as a contact for the following alerts from Energy Sparks.</p>
 <% if @school.school_admin.any? %>
```

Comment:
Could this just be a check for some unsubscribe emails? This way there's logic for them being from the school admin in two places.

Reply:
Yep, also good spot - best to keep school.school_admin out of it..